firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -116,19 +116,35 @@ class ActivityStreamTest: FeatureFlaggedTestBase {
         navigator.goto(TabTray)
         navigator.performAction(Action.OpenNewTabFromTabTray)
         let topSitesCells = app.collectionViews.links["TopSitesCell"]
-        waitForExistence(topSitesCells.staticTexts[newTopSite["bookmarkLabel"]!], timeout: TIMEOUT_LONG)
+        if #available(iOS 16, *) {
+            waitForExistence(topSitesCells.staticTexts[newTopSite["bookmarkLabel"]!], timeout: TIMEOUT_LONG)
+        } else {
+            waitForExistence(topSitesCells.staticTexts["Mozilla — Internet for people, not profit"], timeout: TIMEOUT_LONG)
```

Comment:
For iOS 15, the dash character used is a different unicode from the one used in other platforms. (Hard to tell by naked eye.)